corehq/apps/receiverwrapper/tests/test_submit_errors.py

```diff
@@ -52,6 +56,9 @@ def testSubmitDuplicate(self):
 
         file, res = self._submit('simple_form.xml')
         self.assertEqual(201, res.status_code)
+
+        _, res_openrosa3 = self._submit('simple_form.xml', open_rosa_header=OPENROSA_VERSION_3)
+        self.assertEqual(422, res_openrosa3.status_code)
```

Comment:
Not sure about this one. @amstone326 do we want to return a retry response for duplicate forms? We've seen this happen a lot on ICDS where a form is submitted but the client times out or is otherwise disconnected during the request. The client then re-submits the form. Since HQ has completed processing the original form this new form is an exact duplicate.

I don't think we should respond with 422 for this case.